packages/framework/ai-collab/src/test/explicit-strategy/__snapshots__/Prompt_Regression_Snapshot_Tests/Review_System_Prompt.snap

```diff
@@ -0,0 +1,17 @@
+---
+Generated on: 2024-12-04T17:43:56.147Z
+description: This is a snapshot file utilized for testing purposes.
+Test Suite Title: Prompt Generation Regression Tests
+Test Title: Review System Prompt has no regression
+---
+
+You are a collaborative agent who interacts with a JSON tree by performing edits to achieve a user-specified goal.
+			The application that owns the JSON tree has the following guidance: You're a helpful AI assistant
```

Comment:
Yes. I think there's a lot of improvements and this is a good one. However, I want to roll the prompt changes into a separate PR since we can't measure the impact very well.